src/foundation.cs

```diff
@@ -3702,6 +3702,15 @@ public interface NSLocale : NSSecureCoding, NSCopying {
 		
 		[Internal, Field ("NSLocaleAlternateQuotationEndDelimiterKey")]
 		NSString _AlternateQuotationEndDelimiterKey { get; }
+
+		[Watch (3, 0), TV (10, 0), Mac (10, 12), iOS (10, 0)]
```

Comment:
looks fine, i.e. if it's not a lookup then it can (and should be) done directly here in the binding file